src/App.css

```
@@ -0,0 +1,118 @@
+@font-face {
+  font-family: 'Gatwick';
+  src: url('./fonts/Gatwick-Bold.otf');
+}
+
+h1 {
+  text-align: center;
+  font-family: Gatwick;
+  letter-spacing: 0;
+}
+
+h3 {
+  margin: 0;
+}
+
+body, input, button {
+  font-family: Helvetica;
+  letter-spacing: 0.5px;
+  font-size: 16px;
+}
+
+input:focus, button:focus {
+  outline: none;
+}
+
+a {
+  color: black;
+}
+
+.App {
+  margin: 10px 100px;
+}
+
+.article-container--no-results {
+  font-size: 22px;
+  font-weight: bold;
+  text-align: center;
+  margin: 30px 0;
+}
+
+.article {
+  width: 100%;
+  display: flex;
+  margin-bottom: 20px;
+  padding-bottom: 20px;
+  border-bottom: 1px solid black;
+}
+
+.article-clear {
+  margin-bottom: 30px;
+  background: white;
+  border: 1px solid black;
+  padding: 10px;
+  border-radius: 25px;
+}
+
+.article-clear:hover {
+  cursor: pointer;
+  color: white;
+  border: 1px solid #9BA17B;
+  background-color: #9BA17B;
+  transition: 0.5s;
+}
+
+.article-img {
+  width: 40%;
+}
+
+.article-info {
+  width: 60%;
+  margin-left: 10px;
+}
+
+.article-img img {
+  width: 100%;
+}
+
+.article-link {
+  margin-top: 30px;
+  display: block;
+  font-weight: bold;
+  color: #9BA17B;
+}
+
+.article-link:hover {
+  color: black;
+  transition: 0.5s;
```

consider specifying the property to be transitioned. narrowing the scope of properties the browser needs to monitor for changes can positively impact performance - e.g. higher frame rates - when there are numerous or complex transitions.
```suggestion
  transition: color 0.5s;
```

---

ah this is a good catch, and admittedly i don't usually specify my transition properties (although i don't work much with transitions)